### DISCUSSION TITLE
what's the lowest support version on IOS and Android

### DISCUSSION THREAD

**Discussion by HomyeeKing** [nolimits4web/swiper#5809] [Q&A]
the swiper crashed on these devices
```
iphone 6plus ios10.0.2
iPhone 6S iOS 9.0
```
so I wonder what's the lowest compatiable version

**Comment by Jasmine26z** [comment]
the question resolved? I met the problem that the swiper cannot slide in ios 11 & 12

**Comment by HomyeeKing** [comment]
@Jasmine26z  use swiper 8.x would be fine， because 9.x begin to use web component which doesn't have good compatibility


see the changelog for more details https://github.com/nolimits4web/swiper/releases/tag/v9.0.0

**Reply by Jasmine26z** [reply]
Thanks!
